client/package.json

```diff
@@ -10,6 +10,7 @@
     "@testing-library/user-event": "^12.8.3",
     "antd": "^4.14.1",
     "axios": "^0.21.1",
+    "date-fns": "^2.20.1",
```

Comment:
Alaa, we already dealing with date library (moment.js) - I think we don't need to install a new one (date-fns)